LibrarySystem/src/Model/Validators/Logger/ErrorLogger.cs

```diff
@@ -0,0 +1,28 @@
+using Serilog;
+
+namespace LibrarySystem;
+
+public class ErrorLogger
+{
+    public void LogErrorBasedOnSeverity(SeverityLevel severity, string message, Exception ex)
```

Comment:
Would it be possible for this method to be static? Or what about using a design pattern to keep a single instance of this class.

Reply:
I did not enter a message because if not, that message would be printed on the console and it would not look good, and the Log needs a message so I simply entered an empty one if in the future I want to use a message without the format that we put on the console, but if you wonder why I did not put the message inside the Log, I tried it but it did not come out well on the console because the aesthetics of spectre console came out wrong.

Regarding the static, I did take it into account but I kept it that way so as not to be having bad practices and about a pattern, I do not know if it would be very necessary since it is something small of only a few lines of code, also if we do it it will increase the complexity of the code much more.

Reply:
I understand, I think it's possible to print on the console by removing the "WriteTo.Console" [sink](https://github.com/serilog/serilog/wiki/Configuration-Basics#sinks).

Reply:
Done